zme_dummy.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
```

Comment:
What is this file?

Reply:
This file is just for testing of init.d script.

Reply:
Do we need it permanently here?

Reply:
It's complicated question :-). It's better to save it here near other related scripts, than keep it separately.

Reply:
ok